tasks/asserts.yml

```diff
@@ -3,12 +3,26 @@
 ###
 ### Validate required defines
 ###
+- name: Display cluster
+  debug:
+    msg: "{{ cluster }}"
+  tags:
+    - ci_test
+
+- name: Display cluster name
+  debug:
+    msg: "{{ cluster.name }}"
```

Comment:
According to CI, seems like the `.name` key is not available...

Reply:
Yes, but we can see cluster content in line 8. And only failed in CI (include travis and github action). If you run `make test` locally. It will pass.

Reply:
Be that as it may, but it is unacceptable for the test to behave differently, depending on where it is executed.

Reply:
Agree, I still try to fix this CI issue.

Reply:
Update ansible version to 2.8 to fix this index issue